chainer/optimizer_hooks/gradient_lars.py

```diff
@@ -0,0 +1,83 @@
+from chainer import cuda
+
+
+class GradientLARS(object):
+
+    """Optimizer/UpdateRule hook function for layer wise adaptive rate scaling.
+
+    See: `Large Batch Training of Convolutional Networks \
+          <https://arxiv.org/abs/1708.03888>`_
+       : `Convergence Analysis of Gradient Descent Algorithms \
+          with Proportional Updates \
+          <https://arxiv.org/abs/1801.03137>`_
+
+    This hook function scales all gradient arrays to fit to the weight norm.
+
+    In <https://arxiv.org/abs/1708.03888>
+        \\Delta w_{t+1} = m * \\Delta w_t
+                        + \\gamma * \\lambda * (\\nabla L_t(w) + \\beta w_t)
+        where
+           \\gamma : learning_rate
+           m       : momentum
+           \\beta  : weight_decay
+           \\eta   : lars_coeeficient
+           \\lamda : local_lr
+                     = \\eta * \\frac{\|w\|}{\|\\nabla L(w)\| + \\beta * \|w\|}
```

Comment:
I think replacing `w` with `w_t` would be good as the unsubscripted `w` is not used in the previous fomula.

Reply:
Add a period (`.`) after the last formula.